README.md

```diff
@@ -181,6 +187,9 @@ REST API
 
 ## Repair Schedule Resource
 
+* GET     /
```

Comment:
This should be the root or RepairSchedule resource, not the root of the whole host path.